docs/current_docs/api/arguments.mdx

```diff
@@ -267,6 +297,15 @@ The PHP SDK needs the typing information at runtime to correctly report to the A
 ```php file=./snippets/functions/arguments-array/php/src/MyModule.php
 ```
 </TabItem>
+<TabItem value="Java">
+:::note
+You can also use the `java.util.List` interface to represent a list of values.
+For instance instead of the `String[] names` argument in the example, you can have `List<String> names`.
+:::
+
+```java file=./snippets/functions/arguments-array/java/src/main/java/io/dagger/modules/mymodule/MyModule.java
```

Comment:
Error https://dagger.cloud/dagger/traces/6131e753f919fd3d0e183d804e7a85a1

Reply:
this works for me with `main` version